### ISSUE TITLE
Allow the desktop app to be used without the need to provide an email address and a password

### ISSUE THREAD

**Issue by devdan** [wireapp/wire-desktop#108]
If you have installed Wire via phone, you already have an account registered with your number. It should not be necessary to provide more information such as an email address or password. 

Implement linking between phone and desktop via QR code as seen in Signal and  WhatsApp.

**Comment by tokariu** [comment]
I agree that you shouldn't have to enter an email-address when you already registered an account with a phone number.
Actually you shouldn't even need an email-address!
BUT,  QR code is really not a good idea. I hated it very much in Signal and WhatsApp already.

Threema and Tox found a much better solution: a random but unique hash-id as identifier (to replace the email address). This is far more secure and anonymous than an email-address, which is for sure the favorite selector for any blackhat who wants to keep track of one account and over it's borders.

This should be taken serious and the current solution is far from perfect when it is about security and privacy.

I don't know why Wire keeps on using email-addresses as identifier. I guess they want to be able to get in contact with it's users (over email), when there is need for news and password resets maybe. In that case I can only recommend: take a look at how Threema handled/workarounded these problems. They really found a good solution that fits both sides. Why shouldn't we (wire users/devs) make use of similar approved solutions?

**Comment by devdan** [comment]
What exactly is the problem with QR codes? Can you elaborate on this?

The problem with Threema/Tox ids is that people lose those ids on a regular basis. In practise, my Threema contact list is full of multiple keys per person because people will reinstall their phone including Threema and are not able to revoke their old id, so you have to guess which one is the latest. This is very inconvenient and annoying.

While it would be great to have such an id as an identifier for those who want it instead of using a phone number, please do not make it a default because most people don't want to keep/remember anything beside their phone number. 

Again, what's wrong with the Signal/WhatsApp solution?

**Comment by tokariu** [comment]
I got you on that. But that's exactly why Threema tells you to instantly make a backup of your account-profile, when you've registered only with a threema-id. Obviously, as you say, many ppl don't do that even if told to.
The less experienced users won't use this very good feature anyway, they simply stick with their phone number and address books instead.

I don't say the fictional random, unique, hash-id "Wire-ID" should be the only option available. But it should at least -be- available to the users who want it.  Others might still use their phone-number or email-addresses if they want to.

What's wrong with Signal/WhatsApp QR-Code solution?
The very big Problem here is, if you use QR-Codes, you will make Wire dependent on smartphones and Wire-mobile the "main" Wire client. You have to have a device capable of taking pictures (camera) and analyze/get the QR-Code from the desktop monitor.
That's what I already hated on WhatsApp/Signal's solution: it makes the mobile app the main client of all Wire clients. If you want to sign in on another device, you have to use the mobile device to authenticate with the QR-Code.

I really *love* Wire for the reason that you can use it on PC/desktop without any mobile device involved! At the moment it doesn't matter on which device you use Wire, it just works -independently- on each one.
This is really a big feature that takes Wire ahead of all it's competitors and this shouldn't be taken away when using QR-Codes to sign in on other clients.

if QR-Code sign-in is just optional and other methods like email/password, wire-id/password, phone-number/sms-verificaton still works - THEN it might be okay. Otherwise I would be really really disappointed.

**Comment by devdan** [comment]
Okay, totally got your point about wanting a desktop client independent from a mobile device. 

But making the QR code optional would make everybody happy, I guess.

**Comment by teller** [comment]
Old thread but a response from the Wire team side - this is unlikely to change from the way Wire works today.

The email is there as a 2ndary measure in case you lose access to your phone number. In such case having email connected to your account allows you to still access your account on desktop and recover chats and assets.

**Comment by darkbluelight** [comment]
@teller: The Wire team should really reconsider providing a "Threema like" experience to privacy oriented users. Not having this available in Wire is the main reason I still didn't uninstall Threema.

>The email is there as a 2ndary measure in case you lose access to your phone number.

This is only useful for people that provided their phone number in the first place. Obviously Threema is going quite well without the need to provide private info such as the email or the phone number.

Suggestion: When generating a new account, do it like this:
Step 1) Generate a Wire-ID, like Threema does
Step 2) Ask the user how he wants to backup his account:
- Provide his email-address
- And / or his phone number
- And / or save his "private keys" by file like Threema does or IMHO even better like most Bitcoin Wallets do by saving "the seed" which are 12 to 24 words

If you implement this I predict you get a good percentage of additional Threema users that switch.
